browser/react/components/Login/Login.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+export default (props) => (
+  <div style={props.styles.container}>
+    <div>
+      <Link to="signup" style={{ 'textDecoration': 'none' }}>
+        <button key="signup" style={props.styles.signupButton}>Sign Up</button>
+      </Link>
+    </div>
+    <div style={props.styles.orDividerLineDiv}>
+      <div style={props.styles.orDividerLineBefore}></div>
+      <p style={props.styles.orDivider}></p>
+      <div style={props.styles.orDividerLineAfter}></div>
+    </div>
+    <form onSubmit={props.login}>
+      <div className="form-group">
+        <input
+          key="name"
+          name="email"
+          type="email"
+          placeholder="email"
+          style={props.styles.formControl}
+          required
+        />
+      </div>
+      <div className="form-group">
+        <input
+          key="password"
+          name="password"
+          type="password"
+          placeholder="password"
+          style={props.styles.formControl}
+          required
+        />
+      </div>
+      <button style={props.styles.loginButton} type="submit">Log In</button>
+    </form>
+    <div style={props.styles.orDividerLineDiv}>
+      <div style={props.styles.orDividerLineBefore}></div>
+      <p style={props.styles.orDivider}>or</p>
+      <div style={props.styles.orDividerLineAfter}></div>
+    </div>
+    <div>
+      <a target="_self" href="/api/auth/google/login" style={props.styles.loginWithGoogle}>
+        <span className="fa fa-google" style={props.styles.loginWithGoogleIcon}></span>
+        Log in with Google
+        </a>
+    </div>
+    <a href="#" onClick={props.logout} style={props.styles.logout}>Log Out</a>
```

Comment:
Is there any reason to have this?  There's no way to ever access this link once a user is logged in.